gui/default/syncthing/core/syncthingController.js

```diff
@@ -1192,6 +1192,61 @@ angular.module('syncthing.core')
             return '?';
         };
 
+        $scope.rdConnType = function(deviceID){
+            var conn = $scope.connections[deviceID];
+            if(!conn)
+                return -1;
+                
+            if (conn.type.startsWith('relay'))
```

Comment:
```suggestion
            if (conn.type.indexOf('relay') === 0)
```

This makes the code work in IE11 and prevents error log spam in the browser (ref: https://github.com/syncthing/syncthing/pull/8097).